v3.go

```diff
@@ -74,3 +76,21 @@ func GetTaskV3(ctx context.Context, client *http.Client) (*TaskMetadataV3, error
 
 	return taskMetadata, nil
 }
+
+// Retrieve ECS Container Metadata in V3 format
+func GetContainerV3(ctx context.Context, client *http.Client) (*ContainerMetadataV3, error) {
+	metadataUrl := os.Getenv(ecsMetadataUriEnvV3)
```

Comment:
Would this need a different URL for fetching container data or is it the same response with different things we can marshal out of it?

Reply:
I believe it doesnt because `ECS_CONTAINER_METADATA_URI` is the "root" for all endpoints. https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v3.html

In the `task` call, we use `fetch` with `fmt.Sprintf("%s/task", metadataUrl)`.
In the `container` call, we use `fetch` on the root which the docs describe as:
> This path returns metadata JSON for the container.

Hope this helps and should answer for both questions.